var/spack/repos/builtin/packages/visit/package.py

```diff
@@ -153,7 +153,8 @@ class Visit(CMakePackage):
 
     # VisIt uses Silo's 'ghost zone' data structures, which are only available
     # in v4.10+ releases: https://wci.llnl.gov/simulation/computer-codes/silo/releases/release-notes-4.10
-    depends_on("silo@4.10: +shared", when="+silo")
+    # Silo versions < 4.11 do not build successfully with Spack
+    depends_on("silo@4.11: +shared", when="+silo")
```

Comment:
To me this means we should fix building `silo@4.10` with Spack, rather than overconstraining `visit`